src/server/session.ts

```diff
@@ -1904,6 +1905,71 @@ export class Session<TMessage = string> implements EventSender {
         });
     }
 
+    private mapCode(args: protocol.MapCodeRequestArgs): protocol.FileCodeEdits[] {
+        const formatOptions = this.getHostFormatOptions();
+        const preferences = this.getHostPreferences();

```

Comment:
Is there a test for checking that mapped code has the right project formatting settings applied?